### DISCUSSION TITLE
Appointment Editor "recurrence"

### DISCUSSION THREAD

**Discussion by Stuntflyer** [mikeberger/borg_calendar#190] [Ideas]
I'm having an issue when working on the recurrence feature, specifically the calendar. The lower section of the calendar is hidden behind the taskbar. Is there any way to see the entire calendar? I place the application window at half the display size against the side, top and bottom of the monitor.

27" 2760x1440 monitor using this native resolution. Win 10 22H2

Mike

**Comment by mikeberger** [comment]
Technically, this is a bug in Java as it will properly adjust the popup location with respect to the sides and bottom of the screen, but not the task bar.  Java does know about the taskbar but ignores it in this case.

I found some code on the internet that should fix this for single monitors or when using Borg on the tallest monitor. I'll create an issue.

The obvious workaround is to move the window away from the task bar or to undock the appointment editor when needed. I've never noticed this as my window is away from the task bar.

**Comment by Stuntflyer** [comment]
Thanks, Mike,

I will move the window away from the task bar until a fix is available.

**Comment by mikeberger** [comment]
You can try this version: https://sourceforge.net/projects/borg-calendar/files/drops/BorgCalendar-1.10.04-2.msi/download
